file.go

```diff
@@ -0,0 +1,42 @@
+package python
+
+/*
+#include <stdio.h>
+#include "go-python.h"
+
+PyObject*
+_gopy_PyFile_FromFile(int fd, char *name, char *mode) {
+    FILE *f = fdopen(fd, mode);
+    PyObject *py = PyFile_FromFile(f, name, mode, NULL);
+    PyFile_SetBufSize(py, 0);
+    return py;
+}
+
+*/
+import "C"
+
+import (
+	"os"
+)
+
+// FromFile converts a Go file to Python file object.
+// Calling close from Python will not close a file descriptor.
```

Comment:
this sounds bad and a likely source of file descriptors leak.

couldn't we use the C callback to close the Go file?
I know it's a bit complicated (and subject to other restrictions b/c of the CGo-1.6 rules) but...

Reply:
The function only allocates an object for Python with the same FD. Since Go program was opening that FD in the first place, I think it's logical for Go to close it.

The only real effect is that Python cannot communicate EOF thru this FD. At the same time, if a user expects an EOF after Python script executing, the FD can be closed directly after executing the script to trigger EOF.